codacy-ios/CodacyTestFeature/CodacyTestFeatureViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CodacyTestFeatureViewController.swift
+//  codacy-ios
+//
+//  Created by Mirza Muharemovic on 14/10/2020.
+//  Copyright © 2020 Mirza Muharemovic. All rights reserved.
+//
+
+import UIKit
+
+class CodacyTestFeatureViewController: UIViewController {
+    var x = 0
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/mmuharemovic/codacy-test/pullRequest?prid=6357619)